src/srf/boolean.cpp

```diff
@@ -63,11 +66,24 @@ SCurve SCurve::MakeCopySplitAgainst(SShell *agnstA, SShell *agnstB,
 
     // First find any vertex that lies on our curve.
     List<SInter> vertpts = {};
-    if(agnstA)
-        FindVertsOnCurve(&vertpts, this, agnstA);
-    if(agnstB)
-        FindVertsOnCurve(&vertpts, this, agnstB);
-
+    if(isExact) {
+        // get a bounding box for our curve
+        Vector cmax = Vector::From(-1e10, -1e10, -1e10);
+        Vector cmin = Vector::From(1e10, 1e10, 1e10);
+        for(int i=0; i<=exact.deg; i++) {
+            cmax.x = max(cmax.x, exact.ctrl[i].x);
```

Comment:
Just today I looked at very similar code somewhere else... There is function to do this GetMaxMin...something I inlined it...